assets/.babelrc

```diff
@@ -1,5 +1,17 @@
 {
-    "presets": [
-        "env"
-    ]
-}
\ No newline at end of file
+  "presets": ["@babel/preset-env"],
+  "env": {
+    "test": {
+      "presets": [
+        [
+          "@babel/preset-env",
+          {
+            "targets": {
+              "node": "8"
+            }
+          }
+        ]
+      ]
+    }
+  }
```

Comment:
@raygesualdo Mind explaining the setup here (I know you did before :))?  Can it be trimmed down?

Reply:
Also, I noticed after merging your branch, the `async` in my `view` tests started failing on travis.  Do we have to set the node version?

Reply:
Shoot, I didn't realize that Travis was set up to go all the way back to node v6. Is that a hard requirement @chrismccord? Note the end of life for Node.js v6 is 12 days from now at the end of the month. If we configure Travis for v8 (an LTS release), it will only affect users who are doing development work on this repo. It will not affect the end users of the library. One of the benefits of v8 is that we don't need to include extra polyfills in our tests because v8 has all the JS features we need.

@snewcomer To your question, this babel setup says "when running in a test environment (Jest), only transpile what's needed for Node.js v8 to run it."